src/Zicht/Bundle/HtmldevBundle/Resources/config/services.xml

```diff
@@ -32,6 +32,10 @@
             <tag name="twig.extension"/>
         </service>
 
+        <service id="Zicht\Bundle\HtmldevBundle\Twig\ValidateExtension" class="Zicht\Bundle\HtmldevBundle\Twig\ValidateExtension">
```

Comment:
Alle andere services zijn nog met een goeie oude ID string gedefineerd. Misschien hier ook doen?

Reply:
Heb het juist bewust zo gedaan, waarom de oude variant aanhouden als we weten dat we die in de toekomst gaan vervangen.